public/walkthroughs/my-custom-walkthrough/walkthrough.adoc

```diff
@@ -0,0 +1,49 @@
+= A custom walkthrough
+
+A custom walkthrough that shows users how to do something.
+
+This walkthrough is so super awesome, wow! Everyone will learn so much from it.
+
+== First task
+
+This is the first task, super duper
+
+=== This is the first step
+
+The first step needs you to do 
+
+[type=verification]
+- Check this thing.
+- Check that thing
+
+=== This is the second step
+
+The second step needs you to do this other thing.
+
+[type=verification]
+Check that thing.
+
+[type=verificationFail]
+Breathe. Calm down. Try again. Contact your admin.
+
+[type=verificationSuccess]
```

Comment:
@david-martin @finp Does thing like like an ok idea to you guys?

If a `verification` has a `verificationFail` or `verificationSuccess`  anywhere between itself and the next `verification` then it will be shown when a user selects `yes/no` in the Radio box?

Reply:
@aidenkeating  I'm not sure what the point of [type=verificationSuccess] is?
If you succeed, your reward is indicated at bottom of screen with one of required bulbs lighting up (and all the bulbs must be lit for you to click next). But this type is optional, right? 
(apologies for the skeuomorphic design mentions)

Reply:
Is there a need for a success message?
What would that be in different cases, or would it always be the same?

Reply:
@finp It is optional, yes. This was mostly added in because it was just the inverse of an error message, so it was pretty simple to add. I can remove it completely if it's something we won't need.

Reply:
@david-martin @finp @aidenkeating We should probably remove it. I can't really think of a use case for it. I had removed all handling of successverifications from the UI code a while back to simplify the code in task.js.